bakerydemo/base/models.py

```diff
@@ -159,14 +159,126 @@ class HomePage(Page):
         related_name='+',
         help_text='Homepage image'
     )
+    hero_text = models.CharField(
+        max_length=255,
+        help_text='Write an introduction for the bakery'
+        )
+    hero_cta = models.CharField(
+        verbose_name='Hero CTA',
```

Comment:
Ah! Makes sense. Probably used in many U.S. design houses and I just haven't come across it in .edu. Yeah the verbose text should clarify just fine.